### ISSUE TITLE
[Platform] For scheduled backups if the frequency is very low, subsequent backup starts and hangs

### ISSUE THREAD

**Issue by VijiYB** [yugabyte/yugabyte-db#8447]
If the frequency of the scheduled backups is very low (ie before the first backup completes next backup starts), the subsequent backups process starts and all run simultaneously causing error.

The task history shows as backup failure but the backup tab shows as 'inprocess' and keeps spinning 

<img width="2013" alt="Screen Shot 2021-05-13 at 11 30 16 PM" src="https://user-images.githubusercontent.com/78048211/118231777-6928f600-b444-11eb-9094-ddc9c1e602c8.png">

Task history:
<img width="1838" alt="Screen Shot 2021-05-13 at 11 39 45 PM" src="https://user-images.githubusercontent.com/78048211/118231875-8d84d280-b444-11eb-963e-a5cd8adfd47c.png">

Task details shows empty log
<img width="1197" alt="Screen Shot 2021-05-13 at 11 41 30 PM" src="https://user-images.githubusercontent.com/78048211/118232015-c91f9c80-b444-11eb-9097-30946f5a746a.png">

**Comment by VijiYB** [comment]
3 issues here
1. Subsequent backups should not until the previous backup finishes
2. Backup status update incorrect between Task history (shows failed) and Backup tab(shows inprocess)
3. Task history -> see details -> doesnt show anything

**Comment by streddy-yb** [comment]
@kkg-yb @ymahajan - There are at least three different GH issues (this one, #8456, and 7754) tracking related problems with scheduling backups with unrealistic schedules. It will be great to take a step back and define a good default behavior.

**Comment by ymahajan** [comment]
@streddy-yb, @VijiYB yes, i am working on consolidating all these issues and other usability issues around backup. I plan to create a new jira with proper definition and expected behavior.

**Comment by kkg-yb** [comment]
@streddy-yb understood. @ymahajan, once you have the document ready, lets have a call and discuss this.

**Comment by kkg-yb** [comment]
@VijiYB , @streddy-yb  please assign all of the jira items related these and usability issues around backup to me Once @ymahajan  has the doc ready, I will work on getting the items prioritised and fixed.

**Comment by SergeyPotachev** [comment]
Another one issue from the same area - #8676. Maybe a duplicate but unable to find the proper issue. Simply adding it here.
